tests/unit_tests/config/test_queue_config.py

```diff
@@ -331,3 +332,76 @@ def test_wrong_max_submit_raises_validation_error(max_submit_value, error_msg):
         f.write(f"MAX_SUBMIT {max_submit_value}\n")
     with pytest.raises(ConfigValidationError, match=error_msg):
         ErtConfig.from_file("file.ert")
+
+
+@pytest.mark.usefixtures("use_tmpdir")
+@pytest.mark.parametrize(
+    "queue_system, key, value",
+    [
+        ("LSF", "MAX_RUNNING", 50),
+        ("SLURM", "MAX_RUNNING", 50),
+        ("TORQUE", "MAX_RUNNING", 50),
+        ("LSF", "SUBMIT_SLEEP", 4.2),
+        ("SLURM", "SUBMIT_SLEEP", 4.2),
+        ("TORQUE", "SUBMIT_SLEEP", 4.2),
```

Comment:
I assume we are testing negative values somewhere else, and this test focuses only on values being set?
That might be the drivers responsibility I guess. But implementing generic options would require n-implementations of said checks.

Reply:
Yes testing that these values are correct is already tests but it might be a good idea to add another test specifically when setting generic queue options.